docs/terminal/amberleyoakey.md

```diff
@@ -4,7 +4,127 @@
+| Amberley TCU   | Amberley Approach   | 126.200        | AM_APP                                   |
+| Oakey TCU   | Oakey Approach | 125.400        | OK_APP                                   |
+
+
+
+## Airspace
+### AM TCU
+#### Restricted Areas
+By Default YAMB owns all of the R625 Restricted Areas, detailed below:
+
+- R625A (`A015`-`A085`)  
+- R625B (`A025`-`A085`)  
+- R625C (`A045`-`A085`)  
+- R625D (`A085`-`F210`)  
+
+### Optional, as required
+AM APP can negotiate further airspace releases from surrounding ENR sectors to facilitate planned military operations of the following Restricted Areas:
+
+- R650A (`A050`-`A100`)  
+- R650B (`A050`-`F600`)  
+- R662A (`A050`-`F600`)  
+- R662B (`A050`-`F600`)  
+- R671A (`F125`-`F245`)  
+- R671B (`F240`-`F600`)
+
+!!! note
+    See [VATPAC Notams](https://vatpac.org/publications/notam){target=new} for active NOTAMs which may affect military operations.
+
+#### Diagram
+
+<figure markdown>
+![AM TCU Airspace Diagram](img/amtcu.png){ width="952" }
+  <figcaption>AM TCU Airspace Diagram</figcaption>
+</figure>
+
+### AM ADC
+AM ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. 
+
+### OK TCU
+#### Restricted Areas
+By Default YBOK owns all of the R625 Restricted Areas, detailed below:
+
+- R654A (`SFC`-`A065`)  
+- R6545B (`SFC`-`A065`)  
+- R654C (`A065`-`F125`)  
+- R654D (`A065`-`F125`)  
+
+#### Diagram
+
+<figure markdown>
+![OK TCU Airspace Diagram](img/oktcu.png){ width="1060" }
+  <figcaption>OK TCU Airspace Diagram</figcaption>
+</figure>
+
+### OK ADC
+OK ADC owns the CLass C airspace within 5 DME of the OK VOR from `SFC` to `A025`. This airspace is the circuit area.
+
+
+## YAMB Aerodrome
+### Coded Departures (Fast Jets)
+Visual Departures are commonly requested which are often in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for Amberley. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
+
+
+Aircraft will make a visual departure in the circuit direction and fly overhead YAMB to begin tracking for their initial waypoint.
+
+!!! example
+    WOLF03 was assiged the BYRON7 departure with their intial clearance.  
+    **AM ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
+    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
+
+| Departure | Initial Constraint | Route |
+| --------- | ----------| --------- |
+| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
+
+!!! note
+    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+
```

Comment:
Add note that AM ADC will put the cleared coded clearance in the Global Ops Info Field